workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,180 @@
+
+> **Note:** Parallax scrolling does not always work on mobile devices smartphones. However, you can use media queries to turn off the effect on mobile devices.
+
+We can see how the parallax scrolling works in this example [here](https://codepen.io/samdbeckham/full/OPXPNp).
+
+<a href="https://codepen.io/samdbeckham/full/OPXPNp" target="_blank"><img src="https://cloud-kt149rdr8.vercel.app/ezgif.com-add-text.gif" alt="parallax scrolling"/></a>
+
+^As you scroll, each of the layers moves at different speeds.
+
+## Advantages of Parallax Effect
+
+If you want to make your website amazing, it's not enough to show static elements. You have to amaze your visitors and make them want to explore your site, otherwise they can leave fast. You have to show them some cool things and some short visual surprises. The Parallax effect makes it possible. Moving components on scrolling or on hovering will make them interested.
+You can create something interesting using parallax effect of your choice.
+
+## Getting started
+
+Practically a parallax is composed of more than one layer in parallel, moving along on scroll at different speeds, giving us the feeling that they're at different distances.
+
+### Setup
+
+We will use [Repl.it](https://repl.it) to build this project.
+
+> Repl.it is an online code editor. It's similar to Google Docs, but has some important features that make it much better for typing code than a regular text editor.
+
+Fork this repl and explore! [here](https://repl.it/@Giridharhackclu/parallax-starter#index.html).
+
+It contains three `div` elements with classes `layer l1`, `layer l2` and `layer l3` added with some basic styles. Go ahead! use your creativity and customise the styles.
+
+<a href="https://repl.it/@Giridharhackclu/parallax-starter#index.html"><img src="https://cloud-6z92hbh0u.vercel.app/screenshot_2020-09-22_171732.png" style="{border-radius: 3px;}" alt="#index.html"></a>
+
+For applying parallax effect, the thing we need to do is select the layer and change it's speed while scrolling. That's it!
+
+Let's start doing it!
+
+Go to [script.js](https://repl.it/@Giridharhackclu/parallax-starter#script.js) and add the following <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions">`function`</a>.
+
+```javascript
+function parallax(layer, distance, speed) {
+  const item = document.querySelector(layer)
+  item.style.transform = 'translateY(' + -distance * speed + 'px)'
+}
+```
+
+This `function` has three arguments `layer`- the layer you want to add parallax, `distance`- how much we scroll and `speed`- the required speed change. We will add parallax for multiple layers. That is the reason we made a function to recall it for different layers.
+
+Then we will get the element(layer) required using `querySelector`. You can use any JavaScript DOM selector as your wish. Then translate the `Y-offset` of the element using CSS `transform: translateY();` to change the speed of that layer.
```

Comment:
Not sure what this paragraph is trying to say. To me it looks like they’re instructions and I’m supposed to do something, but I’m not sure what I’m supposed to do. Clarify here maybe

Reply:
Oh I see now that it’s more of a description of the previous code block. In that case, I would keep this as part of the previous paragraph

Reply:
Okay:smile: